docker-compose.yml

```diff
@@ -47,15 +47,90 @@ services:
+            ip link set vni101 master br101 up && \
+
+            bridge vlan add dev br-tenant vid 10 self && \
+            ip link add vni10 type vxlan id 10 local 10.0.0.2 dstport 4789 nolearning && \
+            ip link set vni10 master br-tenant up && \
+            bridge vlan add dev vni10 vid 10 pvid untagged && \
+            bridge link set dev vni10 neigh_suppress on learning off && \
+
+            ip link add vni20 type vxlan id 20 local 10.0.0.2 dstport 4789 nolearning && \
+            ip link set vni20 master br-tenant up && \
+            bridge vlan add dev vni20 vid 20 pvid untagged && \
+            bridge link set dev vni20 neigh_suppress on learning off && \
+
+            ip link add vni30 type vxlan id 30 local 10.0.0.2 dstport 4789 nolearning && \
+            ip link set vni30 master br-tenant up && \
+            bridge vlan add dev vni30 vid 30 pvid untagged && \
+            bridge link set dev vni30 neigh_suppress on learning off && \
+
+            ip link add vni40 type vxlan id 40 local 10.0.0.2 dstport 4789 nolearning && \
+            ip link set vni40 master br-tenant up && \
+            bridge vlan add dev vni40 vid 40 pvid untagged && \
+            bridge link set dev vni40 neigh_suppress on learning off && \
+
+            bridge vlan add dev br-tenant vid 20 self && \
+            ip link add link br-tenant name vlan20 type vlan id 20 && \
+            ip link set vlan20 address aa:bb:cc:00:00:21 && \
+            ip link set vlan20 master blue up && \
+            ip address add 20.20.20.1/24 dev vlan20 && \
+
+            bridge vlan add dev br-tenant vid 30 self && \
+            ip link add link br-tenant name vlan30 type vlan id 30 && \
+            ip link set vlan30 address aa:bb:cc:00:00:31 && \
+            ip link set vlan30 master blue up && \
+            ip address add 30.30.30.1/24 dev vlan30 && \
+
+            bridge vlan add dev br-tenant vid 40 self && \
+            ip link add link br-tenant name vlan40 type vlan id 40 && \
+            ip link set vlan40 address aa:bb:cc:00:00:41 && \
+            ip link set vlan40 master green up && \
+            ip address add 40.40.40.1/24 dev vlan40 && \
+
+            bridge vlan add dev br-tenant vid 50 self && \
+            ip link add link br-tenant name vlan50 type vlan id 50 && \
+            ip link set vlan50 address aa:bb:cc:00:00:51 && \
+            ip link set vlan50 master yellow up && \
+            ip address add 50.50.50.1/24 dev vlan50 && \
+
+            ip link set eth1 up && ip addr flush dev eth1 && \
+            ip link set eth1 master br-tenant up && \
+            bridge vlan add dev eth1 vid 10 && \
+
+            bridge vlan add dev eth1 vid 20 && \
+
+            bridge vlan add dev eth1 vid 40 && \
+
+            ip addr flush dev eth2 && ip link set eth2 master br-tenant up && \
+            bridge vlan add dev eth2 vid 50 && \
```

Comment:
I guess this should change because it needs to be access untagged mode. I guees this is trunk now due to the fact that @venkatmahalingam haven't figured out yet the correct configuration. Right ?

Reply:
Yes. You are right. @venkatmahalingam already raised an issue to fix this. 
#78